### ISSUE TITLE
Remove / Ban user

### ISSUE THREAD

**Issue by craigmaloney** [GetTogetherComm/GetTogether#189]
As an administrator / moderator I would expect the ability to remove / ban a user from my team.

I currently don't have a need for this, but I can see where this might be useful.

**Comment by blackbeerben** [comment]
I agree.  We've had trouble in the past and would like to see this feature added. Thanks.
